turbinia/client.py

```diff
@@ -836,22 +841,52 @@ def format_task_status(
       return '\n{0:s}'.format(msg)
 
     # Build up data
-    report = []
-    requester = task_results[0].get('requester')
-    request_id = task_results[0].get('request_id')
     success_types = ['Successful', 'Failed', 'Scheduled or Running']
     success_values = [True, False, None]
     # Reverse mapping values to types
     success_map = dict(zip(success_values, success_types))
+    # This is used for group ID status
+    requests = defaultdict(dict)
```

Comment:
Just curious, is the defaultdict giving us anything over a normal dict in this case?

Reply:
This makes it easier to define a two dimensional dict.